HISTORY.md

```diff
@@ -25,6 +25,9 @@
  * Add Indicators for Multiobjective optimizers
    ([#285](https://github.com/mlpack/ensmallen/pull/285)).
 
+ * Make Callback traits adaptable for MOO
```

Comment:
Let use multi-objective optimization here, just to make sure everyone knows what MOO means.